apps/theming-designer/tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "strict": false,
     "lib": ["es6", "dom"],
     "types": [],
+    "skipLibCheck": true,
```

Comment:
This had to be enabled due to `esModuleInterop` in the updated version of `markdown-to-jsx` (see changes to `react-docsite-components` later). Most of our other projects already have `skipLibCheck` enabled, which is why they didn't catch this. More details in https://github.com/probablyup/markdown-to-jsx/issues/425. 

I also have a [PR there disabling `esModuleInterop`](https://github.com/probablyup/markdown-to-jsx/pull/426)--hopefully that's accepted so we don't force the handful of `@fluentui/react-docsite-components` to also enable `esModuleInterop` or `skipLibCheck` in their docs packages.

Reply:
PR was merged, so we can update markdown-to-jsx to the latest version and this would no longer be needed (though we might keep it here for consistency with the rest of the repo)

Reply:
Thanks for the note on this @ecraig12345. I'd hope to have this removed from the WC's if it's not needed. IMO, consistency is important, but in this case it comes at the expense of type checking. I'm not sure that we get the value of skipping in our case.

Reply:
The thing is, the amount of value provided by skipLibCheck is quite limited to begin with. Regardless of this setting, typescript will continue to check types in your code against themselves and against any directly-referenced external types. What this setting controls is whether to check the consistency of all directly- *and* indirectly-referenced external types *amongst themselves*, which is much less meaningful/important, and can substantially slow down the build. https://www.typescriptlang.org/tsconfig#skipLibCheck

Reply:
Also IIRC this particular type issue wasn't what caused issues with web-components; that was the react-router types I think?